### PR TITLE
update user manual PIM iOS

### PR DESCRIPTION
**Pull request by BobKircher**
iOS documentation was updated with respect to SSL requirements starting with iOS 12.

### CODE REVIEW

File: user_manual/pim/sync_ios.rst [hunk 1]
```diff
@@ -6,52 +6,40 @@ Calendar
...
+
+.. note:: Beginning with iOS 12 an SSL encryption is necessary. Therefore do **not** disable **SSL**
+  (For this reason a certificate is required at your domain, https://letsencrypt.org/ will do)
+
+
 If it's still not working, have a look at the `Troubleshooting Contacts & Calendar`_
 guide.
 
-.. _Troubleshooting Contacts & Calendar: https://docs.nextcloud.org/server/14/admin_manual/issues/index.html#troubleshooting-contacts-calendar
+.. _Troubleshooting Contacts & Calendar: https://docs.nextcloud.org/server/16/admin_manual/issues/index.html#troubleshooting-contacts-calendar
```
Comment (skjnldsv) [inline user_manual/pim/sync_ios.rst]:
```suggestion
.. _Troubleshooting Contacts & Calendar: https://docs.nextcloud.org/server/stable/admin_manual/issues/index.html#troubleshooting-contacts-calendar
```

**Comment by skjnldsv** [comment]
@nextcloud/ios can someone validate this please? :)

**Comment by BobKircher** [comment]
@skjnldsv should I do anything here ? How do I get the required review ?

**Comment by skjnldsv** [comment]
Well, We need people with iOS devices! Please @nextcloud/ios, review!

**Comment by BernieO** [comment]
I can confirm that the proposed changes do work with iOS when service discovery is working which is the case with the documented configuration for apache and nginx.

**Comment by heaterC** [comment]
This has been working for a long time now (see help forum e.g.). Right now I can confirm it working flawlessly on iOS12 and iOS13 devices for instances of Nextcloud 15 (nginx), 16 (apache) and 17 (nginx).

**Pull request merged**